scripts/release/build.test.js

```diff
@@ -1,26 +0,0 @@
-/* This file is a part of @mdn/browser-compat-data
```

Comment:
No more build test?

Reply:
The build test is to ensure that the build output and the repository JSON exactly match one another, but since we're now introducing a new step to modify the data...well, this test becomes irrelevant.  😛